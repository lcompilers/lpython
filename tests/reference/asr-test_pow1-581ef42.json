{
    "basename": "asr-test_pow1-581ef42",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/test_pow1.py",
    "infile_hash": "a7ce7888e7e99bc753e16822749aaae4ee63641675990b3e2a34de10",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-test_pow1-581ef42.stderr",
    "stderr_hash": "7f7cc07cf6f08f2d953d6e6c1f4892e6379faaf8aa668117234293d0",
    "returncode": 2
}