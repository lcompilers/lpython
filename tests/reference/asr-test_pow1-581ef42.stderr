style suggestion: Could have used '**' instead of 'pow'
 --> tests/errors/test_pow1.py:4:11
  |
4 |     print(pow(x,3))
  |           ^^^^^^^^ '**' could be used instead

semantic error: Arguments do not match for any generic procedure, pow
 --> tests/errors/test_pow1.py:4:11
  |
4 |     print(pow(x,3))
  |           ^^^^^^^^ 
